charts/enterprise/blocky/questions.yaml

```
@@ -113,6 +113,37 @@ questions:
                   type: string
                   required: true
                   default: ""
+  - variable: additionalBootstrapDns
+    group: App Configuration
+    label: Additional Bootstrap DNS
+    schema:
+      type: list
+      show_if: [[ "bootstrapDns", "!=", {} ]]
```

```suggestion
      show_if: [[ "bootstrapDns", "!=", {"upstream": "", "ips": []} ]]
```
This is tested and works